workflow-templates/1gp-ci.yml

```diff
@@ -126,21 +165,24 @@ jobs:
 
       - name: 👩‍🌾 Get orgfarm org 👨‍🌾
         id: setup-org
+        if: ${{ needs.Path-Filtering.outputs.apex-changed == 'true' }}
         uses: ncino/action-org-setup/org-provision-action@v3
         with:
-          capability: "{{REPLACE_REPO_NAME}}"
+          capability: '{{REPLACE_REPO_NAME}}'
```

Comment:
Repo name should be a GitHub env var by default. Forget the exact name for it, but we should be able to use that instead of making people update this file.